examples/developer-tools/src/output-generators/factory_base.ts

```diff
@@ -218,9 +226,35 @@ scriptHeaderGenerator.forBlock['factory_base'] = function (
   block: Blockly.Block,
   generator: CodeHeaderGenerator,
 ): string {
+  const language = generator.getLanguage();
   generator.addHeaderLine(
     `<script src="https://unpkg.com/blockly/blockly_compressed.js"></script>`,
   );
+  generator.addHeaderLine(
+    `<script src="https://unpkg.com/blockly/${language}_compressed.js"></script>`,
+  );
   generator.statementToCode(block, 'INPUTS');
   return '';
 };
+
+generatorStubGenerator.forBlock['factory_base'] = function (
+  block: Blockly.Block,
+  generator: GeneratorStubGenerator,
+): string {
+  const lang = generator.getLanguage();
+  const blockName = block.getFieldValue('NAME');
+  const inputs = generator.statementToCode(block, 'INPUTS');
+  const hasOutput = !!block.getInput('OUTPUTCHECK');
+  const returnIfOutput = `// TODO: Change Order.NONE to the correct operator precedence strength
+  return [code, ${generator.getScriptMode() ? lang + '.' : ''}Order.NONE];`;
```

Comment:
Can you pull `generator.getScriptMode() ? lang + '.' : ''` out into `const orderPrefix`? I find it hard to parse when it's smushed inside the `${}`

Reply:
You also use it on line 253.